python/mxnet/contrib/onnx/_import/import_model.py

```diff
@@ -46,5 +43,5 @@ def import_model(model_file):
     except ImportError:
         raise ImportError("Onnx and protobuf need to be installed")
     model_proto = onnx.load(model_file)
-    sym, params = graph.from_onnx(model_proto.graph)
-    return sym, params
+    sym, arg_params, aux_params = graph.from_onnx(model_proto.graph)
+    return sym, arg_params, aux_params
```

Comment:
This is API breakage. I understand this is necessary because the previous implementation was wrong. But let's try to not merge code before they are ready next time.
@nswamy

Reply:
@piiswrong You were very much tagged in the initial PR, you had every chance to review that section of code looks like you missed as well. I do not understand your concern. No one creates bug intentionally, when bugs are found they are getting fixed.

Reply:
@piiswrong I raised this PR just to check a few things on the CI pipeline. This is the real PR - https://github.com/apache/incubator-mxnet/pull/10118. 

Please kindly give your comments there, I will be closing this PR to avoid any confusion and misunderstanding.

With regard to your concerns. The original code does work. Here is an example that you can try with the latest code on the master - https://github.com/apache/incubator-mxnet/blob/master/example/onnx/super_resolution.py

The current code on master fails for a few models from the onnx model zoo like, resnet50. These changes are to fix that. I understand your concerns and will be mindful in the future.

Reply:
@nswamy 
As far as I remember you merged that PR during my vacation.
I was not deeply involved in the design and implementation of this feature. Since you (partially) designed it and you merged it. You should own it. I'm here to help you but you can't depend on me to do your work.
I'm not saying you created this bug intentionally, but you were certainly careless in mergning it.

Also technically this is not a bug. Its a design failure

Reply:
The bigger issue here is you merged that ONNX PR without test coverage. Not even simple cases like resnet was properly tested. Otherwise you would have found this issue.

Reply:
@piiswrong sorry for merging the code without complete test coverage. The mistake was on our part, I thought I will add the model test coverage in a follow up PR. 

I understand that I should have have got even the model tests in the initial PR itself. I will be mindful of this in the future.